### PR TITLE
Accursed protogen markings (bounty, and modular)

### PR DESCRIPTION
**Pull request by CandleJaxx**
## About The Pull Request

adds in a few markings for 'protogen' styles

## How This Contributes To The Nova Sector Roleplay Experience

it doesn't.. protgens are evil and gross and etc etc

## Proof of Testing

Screenshot_003
Screenshot_004
Screenshot_005
Screenshot_001
Screenshot_002

<details>
<summary>Screenshots/Videos</summary>
  see above
</details>

## Changelog
:cl:
add: We found a bunch of old synthetic casings that were left to rot in a scrapyard out of Eridani. Have them, Frontier Scum.
/:cl:

### CODE REVIEW

File: modular_nova/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm [hunk 1]
```diff
@@ -183,6 +183,21 @@
 	icon_state = "legband"
 	affected_bodyparts = LEG_RIGHT | LEG_LEFT
 
+/datum/body_marking/other/protogenlegs
+	name = "Protogen Leg - Digitigrade"
+	icon_state = "protogen"
+	affected_bodyparts = LEG_RIGHT | LEG_LEFT
```
Comment (GoldenAlpharex) [inline modular_nova/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm:189]: Does this mean that they're forced to be taken together, or can they be taken separately?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Two things:
First, I don't know what happened to your screenshots, but the quality on them is pretty bad, it makes it really hard to actually look at the details in there, because there's so much like- Compression artifacts in them? Is that something you think you could fix?

Second, how well does this play out for synthetics? I don't know much about protogens myself, but I imagine most people that will want to play them will use Synthetics as a base to work with. In that case, how does that look with things like leg options they have (especially since I recall protogens having fur).

File: modular_nova/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm (hunk 1 above)
Comment (CandleJaxx) [reply modular_nova/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm:189]: separately- it's a mark for what's selectable

**Comment by CandleJaxx** [comment]
> Two things: First, I don't know what happened to your screenshots, but the quality on them is pretty bad, it makes it really hard to actually look at the details in there, because there's so much like- Compression artifacts in them? Is that something you think you could fix?
> 
> Second, how well does this play out for synthetics? I don't know much about protogens myself, but I imagine most people that will want to play them will use Synthetics as a base to work with. In that case, how does that look with things like leg options they have (especially since I recall protogens having fur).

first: i can't grab better shots, and i'm away from my pc for a while- but they're set to max colors on diff spectrums to showcase it works

second: yeah this is made w a synthetic. they can choose a mammilian bodytype

File: modular_nova/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply modular_nova/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm:189]: I see, good to know.

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me, and approved by the donator.

**Pull request merged**